pyvista/plotting/colors.py

```diff
@@ -373,18 +348,29 @@
     'w': 'white',
 }
 
-
-PARAVIEW_BACKGROUND = [82 / 255.0, 87 / 255.0, 110 / 255.0]
+color_synonyms = {
+    **color_char_to_word,
+    'aqua': 'cyan',
+    'darkgrey': 'darkgray',
+    'darkslategrey': 'darkslategray',
+    'dimgrey': 'dimgray',
+    'fuchsia': 'magenta',
+    'grey': 'gray',
+    'lightgrey': 'lightgray',
+    'lightslategrey': 'lightslategray',
+    'pv': 'paraview',
+    'slategrey': 'slategray',
+}
 
 
 class Color:
     """Helper class to convert between different color representations used in the pyvista library.
 
-    Many pyvista methods accept :attr:`color_like` parameters. This helper class
+    Many pyvista methods accept :ref:`color_like` parameters. This helper class
```

Comment:
Will have to check if this links correctly, unfortunately I'm a bit skeptical.

Reply:
It doesn't. When looking [here](https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#cross-referencing-python-objects), I think any of `:data:`, `:const:` or `:attr:` would fit?

Reply:
How about `:class:`? It's not entirely correct, because `type(color_like)` is not `type`, but it pretty much serves as a type. If that gets rendered correctly it might be an acceptable compromise, if you agree.

Reply:
Whatever works is acceptable to me :)